### ISSUE TITLE
Convert all withCall to withCalls<Props>

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#1009]
Goal:

Make everything consistent, so there is basically one way of doing things.

Background:

`withCalls` is a relative new addition (replacing `withMulti(withCall(abc), withCall(def), ...)`and actually have some additional properties for types and is slightly more future-proof when changing components. 

Generally the `Props` gets lost with `withCall` and is handled (slightly) better in the case of `withCalls<Props>` - nowhere near perfect, we really would like to have decorators at some point. (But more or less given up on that for now, the TS support is just lacking and types become an absolute nightmare)

TL;DR `withCalls` has _some_ benefits, at the very least when you add more stuff, you can just add it

Actions:

- For all `withCall`, replace it with `withCalls<Props>(...)`
- Adjust props as needed, I've noticed there are places it doesn't even get specified, e.g. https://github.com/polkadot-js/apps/blob/master/packages/ui-reactive/src/Balance.tsx#L12 - no `params` in props, but actually used
